2ls.inc

```diff
@@ -0,0 +1,23 @@
+# tool 
+
+TOOL_BINARY=./2ls-binary
+TOOL_NAME=2LS
+
+# map properties to tool options
+
+declare -A OPTIONS
+OPTIONS["label"]="--k-induction --competition-mode --error-label"
+OPTIONS["unreach_call"]="--k-induction --competition-mode"
+OPTIONS["termination"]="--termination --competition-mode"
+OPTIONS["overflow"]="--k-induction --signed-overflow-check --competition-mode"
+OPTIONS["memsafety"]="--k-induction --pointer-check --memory-leak-check --bound\
+s-check --competition-mode"
```

Comment:
That's possible if $PROP is available in the the run() function so that I can distinguish between safety and termination.